demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -510,7 +533,15 @@ def test_unified_files_ignored(self):
         file_validator.validate_added_files({INVALID_IGNORED_UNIFIED_INTEGRATION})
         assert file_validator._is_valid
 
-    def test_get_error_ignore_list(self, mocker):
+    def test_get_error_ignore_list__file_validator(self, mocker):
+        """
+            Given:
+                - A file path to pack ignore
+            When:
+                - running get_error_ignore_list from file validator
+            Then:
+                - verify that the created ignored_errors list is correct
```

Comment:
there is one in validate manager as well (line 571), I'm keeping this right now only because I think the file_validator deprecation should be done in a different PR